src/openforms/submissions/templates/submission_logs/submission_logs.html

```diff
@@ -0,0 +1,55 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls static %}
+
+{% block extrastyle %}
+
+<link rel="stylesheet" href="{% static "bundles/core-css.css" %}">
+{{ block.super }}
+{% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% translate 'Home' %}</a>
+&rsaquo; <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>
+&rsaquo; <a href="{% url 'admin:submissions_submission_change' original.pk %}">{{ original }}</a>
+&rsaquo; {% trans 'Evaluated Logic Logs'|capfirst%}
+</div>
+{% endblock %}
+
+{% block content %}
+    <h1>{% trans  "Logic logs" %}</h1>
+    {% block field_sets %}
+        {% if not logs_activity %}
+            {% trans "No logic rules for that submission" %}
+        
+        {% else %}
+            {% for log in logs_activity  %}
+                <fieldset class="module aligned collapse collapsed">
+                    <h2>{{ log.timestamp|date:'Y-m-d H:i:s' }} - {% trans 'Logic' %} - </h2>
```

Comment:
I think this would benefit the readability by using the `timesince` template filter: https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#timesince

Then, you typically want to combine that with semantic markup so you can still see the exact timestamp on hover:

```suggestion
                    <h2>
                        <time datetime="{{ log.timestamp.isoformat }}" title="{{ log.timestamp|date:'Y-m-d H:i:s' }}">{{ log.timestamp|timesince }}</time>
                        - {% trans 'Logic' %}
                    </h2>
```

Reply:
I agree, but the problem is that if you add a tag inside the H2 one, it will be disabled if the collapse element is hidden. The only way to display is to add text inside the H2 tag directly I think